src/app/issuer/components/issuer-create/issuer-create.component.html

```diff
@@ -1,149 +1,160 @@
-			label="{{ 'General.name' | translate }}"
-			errorMessage="{{ 'Issuer.enterName' | translate }}"
-			[autofocus]="true"
-		></bg-formfield-text>
-
-		<bg-formfield-text
-			class="u-margin-bottom2x"
-			[control]="issuerForm.rawControlMap.issuer_url"
-			[label]="'Website URL'"
-			errorMessage="{{ 'Issuer.enterURL' | translate }}"
-			[urlField]="true"
-			[fieldType]="'url'"
-			type="url"
-		></bg-formfield-text>
-
-		<bg-formfield-select
-			class="u-margin-bottom2x"
-			[control]="issuerForm.rawControlMap.issuer_email"
-			label="{{ 'General.contact' | translate }} E-Mail"
-			placeholder="{{ 'Issuer.enterValidEmail' | translate }}"
-			[options]="emailsOptions"
-			errorMessage="{{ 'Issuer.enterValidEmail' | translate }}"
-		></bg-formfield-select>
-
-		<bg-formfield-text
-			class="u-margin-bottom2x"
-			[control]="issuerForm.rawControlMap.issuer_description"
-			label="{{ 'General.description' | translate }}"
-			errorMessage="{{ 'Issuer.enterDescription' | translate }}"
-			[multiline]="true"
-		></bg-formfield-text>
-
-		<bg-formfield-select
-			class="u-margin-bottom2x"
-			[control]="issuerForm.rawControlMap.issuer_category"
-			label="{{ 'General.category' | translate }}"
-			placeholder="{{ 'Issuer.enterCategory' | translate }}"
-			[options]="[
-				{
-					label: 'Schule (Grundschulen, weiterführende Schulen, Berufsschulen, Institutionen formaler schulischer Bildung)',
-					value: 'schule'
-				},
-				{
-					label: 'Hochschule (Universitäten. Fachhochschulen, ...)',
-					value: 'hochschule'
-				},
-				{
-					label: 'Andere (Bibliotheken, Museen, FabLabs, Unternehmen, Vereine, ...)',
-					value: 'andere'
-				}
-			]"
-		></bg-formfield-select>
-
-		<h2>{{ 'General.address' | translate }}</h2>
-
-		<bg-formfield-text
-			class="u-margin-bottom2x"
-			[control]="issuerForm.rawControlMap.issuer_street"
-			label="{{ 'General.street' | translate }}"
-			errorMessage="{{ 'Issuer.enterStreet' | translate }}"
-		></bg-formfield-text>
-
-		<bg-formfield-text
-			class="u-margin-bottom2x"
-			[control]="issuerForm.rawControlMap.issuer_streetnumber"
-			label="{{ 'Issuer.houseNo' | translate }}"
-			[fieldType]="'number'"
-			errorMessage="{{ 'Issuer.enterHouseNo' | translate }}"
-		></bg-formfield-text>
-
-		<bg-formfield-text
-			class="u-margin-bottom2x forminput-hide-arrows"
-			[control]="issuerForm.rawControlMap.issuer_zip"
-			label="{{ 'General.postalCode' | translate }}"
-			[fieldType]="'number'"
-			errorMessage="{{ 'Issuer.enterPostalCode' | translate }}"
-		></bg-formfield-text>
-
-		<bg-formfield-text
-			class="u-margin-bottom2x"
-			[control]="issuerForm.rawControlMap.issuer_city"
-			label="{{ 'General.city' | translate }}"
-			errorMessage="{{ 'Issuer.enterCity' | translate }}"
-		></bg-formfield-text>
-
-		<div class="u-margin-top3x">
-			<form>
-				<altcha-widget auto="onfocus" id="altcha"></altcha-widget>
-			</form>
+		<div class="oeb-headline-container-sm">
```

Comment:
The content is a little "centered" in the design.

Reply:
I think they are still working on it, as there are a few new other things.